### PR TITLE
Change fusionVersion Type to String

### PR DESCRIPTION
**Pull request by munishchouhan**
This PR will solve the Jackson deserialization issue of fusionVersion Object

### CODE REVIEW

File: src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy [hunk 1]
```diff
@@ -78,7 +78,7 @@ class WaveContainerRecordTest extends Specification {
         container.timestamp == OffsetDateTime.parse(req.timestamp).toInstant()
         container.zoneId == OffsetDateTime.parse(req.timestamp).offset.id
         container.expiration == exp
-        container.fusionVersion == req.containerConfig.fusionVersion()
+        container.fusionVersion == req.containerConfig.fusionVersion().number
```
Comment (pditommaso) [inline src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy:81]: Can `fusionVersion()` return `null`?

Comment (pditommaso) [reply src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy:81]: then 
```suggestion
        container.fusionVersion == req.containerConfig.fusionVersion()?.number
```

then, it should be like this, no? ☝️

Comment (munishchouhan) [reply src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy:81]: sorry this is a test, where I already added fusion layer
https://github.com/seqeralabs/wave/blob/1f39d0bf8b1c68c6aa73d108c53d6bcd36627526/src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy#L43

Comment (pditommaso) [reply src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy:81]: oops, ok

**Comment by pditommaso** [comment]
Does this change require any update in the Surreal query?

**Comment by munishchouhan** [comment]
> Does this change require any update in the Surreal query?

No, Query only checks if fusionVersion exists in a record or not

**Pull request merged**